Changelog.md

```diff
@@ -11,15 +11,43 @@ Breaking Changes:
 Language Features:
  * General: Allow appending ``calldata`` keyword to types, to explicitly specify data location for arguments of external functions.
 
-Bugfixes:
-
-
+Breaking changes already present in earlier releases as "experimental 0.5.0" (sorted by importance):
```

Comment:
I think this should read something like `via the "experimental 0.5.0" mode, but now they are not optional anymore`.